migrations.go

```diff
@@ -33,20 +28,46 @@ func (m *Migration) String() string {
 	return strconv.FormatInt(m.Version, 10)
 }
 
+// A Group is a collection of migrations supporting registration and execution.
+type Group struct {
+	// DiscoverSQL enables auto-discovery and registration of SQL files as migration steps.
+	DiscoverSQL bool
```

Comment:
The default value here is false, but that could be changed. https://github.com/go-pg/migrations/pull/31 added some pretty complicated and specific automatic behavior, in a way that broke at least our workflow. Defaulting it to off seems like the safest option - the behavior change is recent enough that probably there are not so many folks relying on the current "default on" state yet.